### ISSUE TITLE
NTR: Irradiation exposure history (Cranium & spine)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7805]
New term request Irradiation exposure history (segal_110204133655): Includes spinal, abdominal, pelvic, thoracic  UMLS: 1522449

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000181
- label: History of radiation therapy
- parents: [Past medical history (HP:0032443)](https://hpo.jax.org/app/browse/term/HP:0032443)
- pmid: 
- def: A medical history of having undergone radiation therapy, which is a treatment that uses ionizing radiation for purposes such as the control the growth of malignant cells in cancer treatment.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7805
- synonyms:
